packages/test-helpers/isAddress.ts

```diff
@@ -0,0 +1,3 @@
+export function isAddress(value: string): boolean {
+  return /^(0x)?[0-9a-f]{40}$/i.test(value)
```

Comment:
I don’t think we have addresses without `0x` prefix:

```suggestion
  return /^0x[0-9a-f]{40}$/i.test(value)
```